### ISSUE TITLE
Question about heading levels

### ISSUE THREAD

**Issue by jasmussen** [WordPress/wporg-make#5]
There are a lot of incorrect heading levels on the Make blogs, which becomes a poorly accessible experience for screen reader users. For example, these headings are h1s, they should be h2s:

![Image](https://github.com/user-attachments/assets/0ada507d-123d-4a7a-911d-14973f520abc)

That also means we need an H1 someplace on the page. That could be this one, potentially:

![Image](https://github.com/user-attachments/assets/8cb3b17d-fb85-46f7-831a-9e62b93955d2)

There are many more heading level errors, so it depends what’s feasible to do in the codebase, whether this is something that can be fixed for this first iteration or not.
